.github/workflows/ci.yml

```diff
@@ -391,3 +391,23 @@ jobs:
           fail_ci_if_error: false
           verbose: true
           token: ${{ secrets.CODECOV_TOKEN }}
+  build-release-binaries:
+    # This overwrites the previously built testing binaries with versions without coverage.
+    name: Rebuild binaries
+    needs: [matrix, test-collect-reports]
```

Comment:
I'm wondering if we could have this run in parallel to the test steps to avoid adding additional CI time?  `ci-build-binaries` does an `upload-artifact`, so we need to be careful to not overwrite those, but it might be possible to name the artifact something different for the coverage and non-coverage build?

Reply:
Hmm. I think this would be a good idea considering the ci build time. I should be able to test this in parallel while working on some other issues.

Reply:
Sounds good to me!